version_bump.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+
+# set up arguments
+parser = argparse.ArgumentParser(prog="version bump")
+parser.add_argument('-b', '--bump', default="", help="BUMP should be an integer between 0 and 3" \
+                                                     " indicating the type of version bump.")
+parser.add_argument('-r', '--release', default="", help="RELEASE is expected to be 'y' or 'n', indicating" \
+                                                        " whether or not the SNAPSHOT suffix should be added.")
+args = parser.parse_args()
+
+# retrieve current version
+version_string = open("u8timeseries/VERSION", "r").read()
```

Comment:
for the sake of reusing this script in other projects, could you move u8timeseries to an argument (eg. VERSION_FILE)?

Reply:
Do you mean I should define this as a constant somewhere (at the top of the script), or should I pass it as an argument like -b and -r?

Reply:
the second one :)

Reply:
We could maybe have both? An argument, and a default value (specified at the top of the file in a variable), so that it keeps the ease of use of the script.